src/dashboards/components/dashboard_index/DashboardCardsPaginated.tsx

```diff
@@ -72,30 +88,33 @@ class DashboardCards extends PureComponent<Props> {
     const {pinnedItems} = this.state
 
     return (
-      <div>
```

Comment:
can you make sure this still works with multiple pages of dashboard cards? I seem to remember this being added in to make scrolling work properly

Reply:
Still works as far as i can tell. See video.

Reply:
I added another video that has scrollbars and pagination together.